proto/osmosis/concentrated-liquidity/position.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+// this is a legacy package that requires additional migration logic
+// in order to use the correct packge. Decision made to use legacy package path
+// until clear steps for migration logic and the unknowns for state breaking are
+// investigated for changing proto package.
+package osmosis.concentratedliquidity.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity/types";
+
+message TickInfo {
```

Comment:
nit: consider moving `TickInfo` to a separate file as well. The per-file logical grouping with `Position` doesn't really make sense IMO

Reply:
Created a separate file